query-engine/query-engine-wasm/build.sh

```diff
@@ -28,12 +26,20 @@ then
     curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
 fi
 
+echo "ℹ️  Configuring rust toolchain to use nightly and rust-src component"
+rustup default nightly-2024-01-25 
+rustup target add wasm32-unknown-unknown
+rustup component add rust-src --target wasm32-unknown-unknown
+
+
 echo "Building query-engine-wasm using $WASM_BUILD_PROFILE profile"
-CARGO_PROFILE_RELEASE_OPT_LEVEL="z" wasm-pack build "--$WASM_BUILD_PROFILE" --target $OUT_TARGET --out-dir "$OUT_FOLDER" --out-name query_engine
+export RUSTFLAGS="-Zlocation-detail=none"
```

Comment:
Uhm, does this `RUSTFLAGS` need to be exported?
Couldn't it be defined inline in the `wasm-pack build` command below, like the `-Zbuild-std*` flags?

Reply:
it's a compiler flag, not a cargo flag

Reply:
we could inline it by calling `RUSTFLAGS="..." wasm-pack ...` as a single command but I think it's fine either way

Reply:
Ok thanks, then I also think this is fine :)